ui/packages/catalog-ui-search/src/main/webapp/component/query-settings/query-settings.view.js

```diff
@@ -58,10 +58,14 @@ define([
             this.setupSortFieldDropdown();
             this.turnOnEditing();
 
-            if (ResultForm.getResultTemplatesProperties()) {
+            let resultTemplates = ResultForm.getResultTemplatesProperties();
+            //TODO: Figure out why the All Fields form always shows up at the end and maybe reverse the way the array is constructed
```

Comment:
It's just because `Array.push` adds on to the back and we add `All Fields` as the last element in [result-form.js](https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.js#L47-L53)

Removing the TODO